.github/workflows/ci.yml

```diff
@@ -70,7 +80,9 @@ jobs:
             ${{ runner.os }}-build-
             ${{ runner.os }}-
       - name: Run tests
-        run: crystal spec
+        run: |
+          export SHARDS_OVERRIDE="$(pwd)/shard.override.yml"
```

Comment:
I had to put this here because I couldn't get the dynamic `$(pwd)` to work under the `env` key, and it fails if you call `source` because that utility doesn't exist on this container for some reason (maybe security?).

Reply:
Do you know why this is needed at all? I thought it would use `shard.override.yml` by default? Sounds like it might be a bug in shards

Reply:
It does, but it assumes the file is in the same directory which in this case, it's not. We generate a test project where the `shard.yml` is in `test-project/shard.yml`, but the override is at `shard.override.yml` and not `test-project/shard.override.yml`. So in this case, we export the env var, and when we run `test-project/shards install`, it picks it up. Does that make sense?

Reply:
Ah yeah that makes sense! Thanks for the explanation!